pyanaconda/modules/network/network_interface.py

```diff
@@ -22,8 +22,17 @@
 from pyanaconda.dbus.property import emits_properties_changed
 from pyanaconda.dbus.typing import *  # pylint: disable=wildcard-import
 from pyanaconda.modules.common.base import KickstartModuleInterface
-from pyanaconda.dbus.interface import dbus_interface, dbus_signal
+from pyanaconda.dbus.interface import dbus_interface, dbus_signal, dbus_class
 from pyanaconda.dbus.structure import get_structure
+from pyanaconda.modules.common.task import TaskInterface
+
+
+@dbus_class
+class InitializeTaskInterface(TaskInterface):
```

Comment:
It might be useful also for other tasks then for the initialization. Maybe, I would choose a more specific name like `NetworkTaskInterface` or `ConnectionsTaskInterface`. Also, please add a docstring.

Reply:
I added the docstring and renamed the interface.
I still don't see how it could be used more generally, so I chose the name related to initialization tasks (the tasks return list of device names). I may make it more general it if it makes sense when adding other tasks in the future.